ecs-cli/modules/cli/compose/entity/task/task.go

```diff
@@ -366,6 +367,18 @@ func (t *Task) buildRunTaskInput(taskDefinition string, count int, overrides map
 		runTaskInput.LaunchType = aws.String(launchType)
 	}
 
+	tags, err := t.GetTags()
+	if err != nil {
+		return nil, err
+	}
+	if len(tags) > 0 {
+		runTaskInput.Tags = tags
```

Comment:
Do we also need to worry about setting `PropagateTags` here as in we do on [createServiceInput](https://github.com/aws/amazon-ecs-cli/pull/710/files#diff-499d2f9fc374c259c658f974cbc87b46R539)?